README.ru.md

```diff
@@ -1,5 +1,13 @@
+<div class="hide-in-mkdocs">
+
 # Synthetic Data Values Generator (SDVG)
 
+Документация доступна на 2-ух языках:
```

Comment:
You might have wanted to write this, but it looks very unusual due to its placement in README

```suggestion
Документация доступна в 2-х версиях:
```